src/labels/CustomLabels.labels

```diff
@@ -1942,6 +1942,38 @@
         <value>Enhanced Recurring Donations makes Recurring Donations more flexible and easier to manage than ever before. To enable Enhanced Recurring Donations, follow the steps below. You must complete ALL the steps to fully enable Enhanced Recurring Donations.
         &lt;br/&gt;&lt;br/&gt;&lt;b&gt;Because you have no Recurring Donation records, you don&apos;t need to validate or migrate any data.&lt;/b&gt;
         &lt;br/&gt;&lt;br/&gt;For more information, read the &lt;a href=&quot;https://powerofus.force.com/NPSP_Recurring_Donations&quot; target=&quot;_blank&quot;&gt;Enhanced Recurring Donations documentation&lt;/a&gt;.</value>
+    </labels>
+     <labels>
+        <fullName>RD2_EntryFormDonationSectionHeader</fullName>
+        <categories>Recurring-Donations, Entry-Form</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Recurring Donation Entry Form Donation Information Section Header</shortDescription>
+        <value>Donation Information</value>
+    </labels>
+    <labels>
+        <fullName>RD2_EntryFormDonorSectionHeader</fullName>
+        <categories>Recurring-Donations, Entry-Form</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Recurring Donation Entry Form Donor Information Section Header</shortDescription>
+        <value>Donor Information</value>
+    </labels>
+    <labels>
+        <fullName>RD2_EntryFormHeader</fullName>
+        <categories>Recurring-Donations, Entry-Form</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Recurring Donation Entry Form Header</shortDescription>
+        <value>New Recurring Donation</value>
+    </labels>
+    <labels>
+        <fullName>RD2_EntryFormOtherSectionHeader</fullName>
+        <categories>Recurring-Donations, Entry-Form</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Recurring Donation Entry Form Other Information Section Header</shortDescription>
+        <value>Other Information</value>
```

Comment:
![Screen Shot 2020-05-15 at 6 23 27 PM](https://user-images.githubusercontent.com/57271120/82106990-4224d080-96d9-11ea-96f3-4743a838d221.png)
@sam-knox Please review the 4 new custom labels for the Entry Form title and section headers